examples/generic-st77xx-with-xpt2046.py

```diff
@@ -0,0 +1,36 @@
+import sys
+sys.path.append('.')
+from st77xx import *
+from xpt2046 import *
+
+spi=machine.SPI(
+    1,
+    baudrate=24_000_000,
+    polarity=0,
+    phase=0,
+    sck=machine.Pin(10,machine.Pin.OUT),
+    mosi=machine.Pin(11,machine.Pin.OUT),
+    miso=machine.Pin(12,machine.Pin.IN)
+)
+
+if 0:
+    # with DMA, the repaints seem to be too slow? To be investigated
+    # we seem to be fine performance-wise without DMA with 320x240 anyway
+    import rp2_dma
+    rp2_dma=rp2_dma.DMA(0)
+else: rp2_dma=None
+
+import lvgl as lv
+lv.init()
+
+lcd=St7789(rot=3,res=(240,320),spi=spi,cs=9,dc=8,bl=13,rst=15,rp2_dma=rp2_dma,factor=8)
```

Comment:
You've already added St7789 and Xpt2046 to advanced_demo example - which is good.  
Is there a need for this tiny example? What does it add over the example code in advanced_demo?  
If you agree this does not add much, consider removing it and the revert run.sh.

Reply:
Not that I can find the comment right now, but you requested a minimal example with a button, which is this one. At first, I had no idea how to run `advanced_demo` with rp2 so it was the LVGL test for me. It makes the relatively simple setup stand out (though I understand it might be superfluous) plus there is the `rp2_dma` portion which someone might tune later (it works, but less efficiently than without).